### DISCUSSION TITLE
How can we fetch photos for a Place?

### DISCUSSION THREAD

**Discussion by aaleksandar** [OvertureMaps/data#51] [Discussion]
What are some of the ways we could fetch photos of Places, either directly or cross-linking with Bing or Meta APIs?

**Comment by Cj-Malone** [comment]
Facebook profile images are available at: `https://graph.facebook.com/{id}/picture` eg `https://graph.facebook.com/137838233253699/picture`

**Reply by myerekapan** [reply]
What's the id you referenced here? GERS?

**Reply by Cj-Malone** [reply]
This is the Facebook id, GERS has only just been added.
